sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -74,6 +74,39 @@
 """,
 ).strip()
 
+# Query to fetch idle blocking sessions
+# with only basic sess and conn info
+# not joining with sys.dm_exec_requests
+IDLE_BLOCKING_SESSIONS_QUERY = re.sub(
+    r'\s+',
+    ' ',
+    """\
+SELECT
+    CONVERT(
+        NVARCHAR, TODATETIMEOFFSET(CURRENT_TIMESTAMP, DATEPART(TZOFFSET, SYSDATETIMEOFFSET())), 126
+    ) as now,
+    sess.login_name as user_name,
+    sess.last_request_start_time as last_request_start_time,
+    sess.session_id as id,
+    DB_NAME(sess.database_id) as database_name,
+    sess.status as session_status,
+    lqt.text as statement_text,
```

Comment:
Do you have concerns about this mis-representing the query statement that was actually doing the blocking?

Reply:
@justiniso after some thoughts, plan to leave the field name as `statement_text`. One reason is as we discussed, it's unlike in the time you ran the query, a different query started, finished, and is sleeping again. Another reason is that we can still leverage `session_status == 'sleeping'` to differentiate the running and sleeping activities. For sleeping activities, we know the query text is from `dm_exec_connections.most_recent_sql_handle` not reqs dmv.